### ISSUE TITLE
Wrong placement of antenna

### ISSUE THREAD

**Issue by chssn** [pierr3/VectorAudio#29]
Hi,

When I log on as EGSU_I_TWR, the transmitter defaults to being over Paris rather than Duxford. [afv-map](https://afv-map.vatsim.net/full_map?callsign=EGSU_I_TWR)

When I use the Audio for VATSIM client, the transmitter appears in the correct place.

afv.log
```Thu Jan 26 11:57:27 2023: APISession: couldn't decode station alias: [json.exception.type_error.302] type must be number, but is null
Thu Jan 26 11:57:27 2023: APISession: got 97 station aliases.
Thu Jan 26 11:57:27 2023: APISession: got error from API server getting station transceivers: Response Code 404
Thu Jan 26 11:57:27 2023: APISession: got error from API server getting vccs: Response Code 404
Thu Jan 26 11:57:48 2023: ATCClient: Need to fetch transceivers for station EGKK_TWR
Thu Jan 26 11:57:48 2023: APISession: got 1 station transceivers for station EGKK_TWR.
Thu Jan 26 11:57:48 2023: ATCClient: Receiving new transceivers for station EGKK_TWR
Thu Jan 26 11:59:44 2023: afv::ATCClient: Voice Session Disconnected
Thu Jan 26 11:59:44 2023: afv_native::ATCClient: Disconnected from AFV API Server.  Terminating sessions
Thu Jan 26 12:04:31 2023: APISession: API Token Expires in 3597 seconds
Thu Jan 26 12:04:31 2023: afv_native::ATCClient: Connected to AFV API Server
Thu Jan 26 12:04:31 2023: ATCClient: Need to fetch transceivers for station EGSU_I_TWR
Thu Jan 26 12:04:31 2023: afv::ATCClient: Voice Session Connected
Thu Jan 26 12:04:31 2023: afv::ATCClient: Initialising Speaker Audio...

vector_audio.log
[2023-01-26 12:03:16.752] [VectorAudio] [info] Starting Vector Audio...
[2023-01-26 12:03:17.081] [VectorAudio] [info] Created afv_native client.
[2023-01-26 12:03:17.081] [VectorAudio] [info] Created data file thread

**Comment by chssn** [comment]
Should also add that I'm using the Windows client v0.4.0 and haven't had any issues using it at other UK aerodromes

**Comment by pierr3** [comment]
That's simply because EGSU_I_TWR does not have a defined transceiver in the database. VectorAudio does not support those kind of stations (see #6), and your FE should add a transceiver for all stations to avoid frequency conflicts. Recommend asking your FE to add the frequency!

**Comment by pierr3** [comment]
That being said, you're right that the UI should be way clearer, as you cannot know when this happens otherwise.

**Comment by pierr3** [comment]
You've motivated me to fix this, starting in 0.5.0 airport of the user will be used as the default transceiver in case none is defined. Will be released soon.
